src/getEditorNamespace.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import loadScript from 'load-script';
+
+/* global CKEDITOR */
+
+let promise;
+
+export default function getEditorNamespace( editorURL, namespaceLoaded ) {
```

Comment:
`namespaceLoaded` sounds more like a name for a boolean variable, maybe let's change it to `onNamespaceLoaded` or just `callback`?